doc/Manuals/ig-cs_x-road_6_central_server_installation_guide.md

```diff
@@ -43,52 +43,50 @@ Doc. ID: IG-CS
 | 29.09.2020 | 2.19    | Add instructions for creating database structure and roles manually. | Ilkka Seppälä
 | 19.01.2021 | 2.20    | Add instructions for using an alternative Java distribution. | Jarkko Hyöty
 | 04.02.2021 | 2.21    | Minor updates. | Ilkka Seppälä
+| 16.04.2021 | 2.22    | Update remote database installation instructions. | Jarkko Hyöty
```

Comment:
I added some comments here and there on the different installation guides. Even though comments are usually on a specific installation guide, almost all seem to apply to all 3.... I noticed this only after finishing.